### ISSUE TITLE
TypeError: Cannot read property 'state' of null

### ISSUE THREAD

**Issue by wyukawa** [tagomoris/shib#24]
More than one queries are issued at the same time, SQLITE_BUSY error occurs.

And, TypeError occurs.

* shib log
```
Mon Mar 02 2015 09:30:03 GMT+0900 (JST) [WARN] : Invalid query submitted, { query: '...',
  err: 'SQLITE_BUSY: database is locked' }
Mon Mar 02 2015 09:30:03 GMT+0900 (JST) [ERROR] : Error in app, { [Error: SQLITE_BUSY: database is locked] errno: 5, code: 'SQLITE_BUSY' }
Mon Mar 02 2015 09:30:03 GMT+0900 (JST) [ERROR] : Error: SQLITE_BUSY: database is locked

/.../shib/app.js:439
    res.send(pseudo_status(query.state));
                                ^
TypeError: Cannot read property 'state' of null
    at null.<anonymous> (/.../shib/app.js:439:33)
    at /.../shib/lib/shib/client.js:193:14
    at Statement.<anonymous> (/.../shib/lib/shib/localdiskstore.js:153:17)
Mon Mar 02 2015 09:35:02 GMT+0900 (JST) [INFO] : Starting shib.
```

**Comment by wyukawa** [comment]
add other log

In this case, hiveserver2 full gc often occurs

```
Sun Mar 01 2015 21:35:03 GMT+0900 (JST) [INFO] : Starting shib.
Mon Mar 02 2015 01:00:10 GMT+0900 (JST) [WARN] : Failed to execute statement, { res: 
   { status: 
      { statusCode: 3,
        infoMessages: null,
        sqlState: '08S01',
        errorCode: 2,
        errorMessage: 'Error while processing statement: FAILED: Execution Error, return code 2 from org.apache.hadoop.hive.ql.exec.mr.MapRedTask' },
     operationHandle: null } }
Mon Mar 02 2015 01:00:10 GMT+0900 (JST) [WARN] : Failed to execute statement, { res: 
   { status: 
      { statusCode: 3,
        infoMessages: null,
        sqlState: '08S01',
        errorCode: 2,
        errorMessage: 'Error while processing statement: FAILED: Execution Error, return code 2 from org.apache.hadoop.hive.ql.exec.mr.MapRedTask' },
     operationHandle: null } }
Mon Mar 02 2015 01:00:11 GMT+0900 (JST) [WARN] : Failed to execute statement, { res: 
   { status: 
      { statusCode: 3,
        infoMessages: null,
        sqlState: '08S01',
        errorCode: 2,
        errorMessage: 'Error while processing statement: FAILED: Execution Error, return code 2 from org.apache.hadoop.hive.ql.exec.mr.MapRedTask' },
     operationHandle: null } }
Mon Mar 02 2015 01:00:11 GMT+0900 (JST) [WARN] : Failed to execute statement, { res: 
   { status: 
      { statusCode: 3,
        infoMessages: null,
        sqlState: '08S01',
        errorCode: 2,
        errorMessage: 'Error while processing statement: FAILED: Execution Error, return code 2 from org.apache.hadoop.hive.ql.exec.mr.MapRedTask' },
     operationHandle: null } }

/.../shib/app.js:439
    res.send(pseudo_status(query.state));
                                ^
TypeError: Cannot read property 'state' of null
    at null.<anonymous> (/.../shib/app.js:439:33)
    at /.../shib/lib/shib/client.js:193:14
    at Statement.<anonymous> (/.../shib/lib/shib/localdiskstore.js:153:17)
Mon Mar 02 2015 01:05:04 GMT+0900 (JST) [INFO] : Starting shib.
```

**Comment by wyukawa** [comment]
This issue seems to be fixed by #31
